stylesheets/_font_stack.scss

```diff
@@ -10,10 +10,12 @@
 //  the font-stack in a font-face declaration rather
 //  than with the usual font-family.
 
+// scss-lint:disable NameFormat
```

Comment:
I've added reasons for each, having a context for each disabled linter is really useful.